packages/flutter/lib/src/gestures/multidrag.dart

```diff
@@ -168,7 +168,8 @@ abstract class MultiDragGestureRecognizer<T extends MultiDragPointerState> exten
   void acceptGesture(int pointer) {
     assert(_pointers != null);
     T state = _pointers[pointer];
-    assert(state != null);
+    if (state == null)
```

Comment:
add a comment saying why it might be null (presumably because it's not in the list, maybe because it's been cancelled already?)